packages/data/src/world/mq/bottom_of_the_well_mq.yml

```diff
@@ -59,7 +41,108 @@
     "MQ Bottom of the Well Wonder Item Main Room Right 2": "can_use_slingshot"
     "MQ Bottom of the Well Wonder Item Main Room Right 3": "can_use_slingshot"
     "MQ Bottom of the Well Wonder Item Main Room Right 4": "can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 1": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 2": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 3": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
-    "MQ Bottom of the Well Wonder Item Side Room 4": "(can_play_zelda || (has_explosives_or_hammer && has_hover_boots)) && has_lens && can_use_slingshot"
+"Bottom of the Well Main Room Center Switch Corner":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main": "has_explosives_or_hammer"
+    "Bottom of the Well Main Room Center": "has_hover_boots || trick(OOT_BOTW_MQ_BACKFLIP) || hookshot_anywhere || climb_anywhere"
+    "Bottom of the Well Basement": "true"
+  events:
+    BOTW_CENTER_WEST_TORCH: "true"
+"Bottom of the Well Main Room Center":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Wallmaster Main": "soul_wallmaster"
+    "Bottom of the Well Main Room Center Switch Corner": "has_hover_boots || trick(OOT_BOTW_MQ_BACKFLIP) || hookshot_anywhere || climb_anywhere"
+    "Bottom of the Well West Center Room": "event(BOTW_CENTER_WEST_TORCH)"
+    "Bottom of the Well East Center Room": "event(BOTW_CENTER_EAST_TORCH)"
+    "Bottom of the Well Basement Switch Ledge": "event(BOTW_MQ_GATE_FROM_SWITCH)"
+    "Bottom of the Well Basement": "true"
+  events:
+    BOTW_CENTER_EAST_TORCH: "can_hit_triggers_distance || can_hookshot || has_explosives || can_use_mask_blast"
+  locations:
+    "MQ Bottom of the Well Map Chest": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 1": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 2": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 3": "true"
+    "MQ Bottom of the Well Lobby Cage Big Fairy": "can_play_sun"
+"Bottom of the Well Basement":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main": "true"
+    "Bottom of the Well Basement Switch Ledge": "climb_anywhere || hookshot_anywhere"
+    "Bottom of the Well Wallmaster Basement": "soul_wallmaster"
```

Comment:
Does going straight to the Pits room with climb anywhere work for MQ?

Reply:
no, as you said above the gate is double-sided collision. would not work.